imap_processing/tests/test_cli.py

```diff
@@ -165,6 +165,39 @@ def test_hi_l1(mock_instrument_dependencies, data_level, n_prods):
         assert mocks["mock_upload"].call_count == n_prods
 
 
+@mock.patch("imap_processing.cli.quaternions.process_quaternions")
```

Comment:
I recommend adding `autospec=True` here. That will cause the mock function to have the same signature as the mocked function and thus the signature of the call will also be tested.
```suggestion
@mock.patch("imap_processing.cli.quaternions.process_quaternions", autospec=True)
```

Reply:
Interesting, I didn't realize that, thank you for that great suggestion!